src/main/java/frc/robot/subsystems/climber/commands/Pull.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.climber.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import frc.robot.Constants;
+import frc.robot.subsystems.climber.Climber;
+
+public class Pull extends CommandBase {
+
+    private final Climber climber;
+    private double height;
+
+    public Pull(Climber climber, double height) {
+        this.climber = climber;
+        this.height = height;
+        addRequirements(climber);
+    }
+
+    @Override
+    public void initialize() {
+        if (climber.getStopperMode() == Climber.PistonMode.OPEN)
+            new SwitchMechanicalStopper(climber);

```

Comment:
You can't call commands like that, you should use Command Group, I don't think you ever tried this, so ask @orel-david